bootstrap/sql/postgresql/v004__set_name_to_all_foreign_keys.sql

```diff
@@ -0,0 +1,90 @@
+SELECT dropForeignKeysFromTable('topology_window');
+SELECT dropForeignKeysFromTable('service');
+SELECT dropForeignKeysFromTable('service_configuration');
+SELECT dropForeignKeysFromTable('component');
+SELECT dropForeignKeysFromTable('component_process');
+SELECT dropForeignKeysFromTable('role_hierarchy');
+SELECT dropForeignKeysFromTable('user_role');
+SELECT dropForeignKeysFromTable('topology_editor_toolbar');
+SELECT dropForeignKeysFromTable('topology_test_run_case');
+SELECT dropForeignKeysFromTable('topology_test_run_case_source');
+SELECT dropForeignKeysFromTable('topology_test_run_case_sink');
+SELECT dropForeignKeysFromTable('topology_test_run_histories');
+
+-- add foreign keys with explicit symbol
+ALTER TABLE widget ADD CONSTRAINT "fk_widget_dashboard" FOREIGN KEY ("dashboardId") REFERENCES dashboard("id");
+ALTER TABLE datasource ADD CONSTRAINT "fk_datasource_dashboard" FOREIGN KEY ("dashboardId") REFERENCES dashboard("id");
+ALTER TABLE widget_datasource_mapping ADD CONSTRAINT "fk_widget_datasource_mapping_widget" FOREIGN KEY ("widgetId") REFERENCES widget("id");
+ALTER TABLE widget_datasource_mapping ADD CONSTRAINT "fk_widget_datasource_mapping_datasource" FOREIGN KEY ("datasourceId") REFERENCES datasource("id");
+ALTER TABLE topology ADD CONSTRAINT "fk_topology_topology_version" FOREIGN KEY ("versionId") REFERENCES topology_version("id");
+ALTER TABLE topology ADD CONSTRAINT "fk_topology_namespace" FOREIGN KEY ("namespaceId") REFERENCES namespace("id");
+ALTER TABLE topology_editor_metadata ADD CONSTRAINT "fk_topology_editor_metadata_topology_version" FOREIGN KEY ("versionId") REFERENCES topology_version("id");
+ALTER TABLE topology_stream ADD CONSTRAINT "fk_topology_stream_topology_version" FOREIGN KEY ("versionId") REFERENCES topology_version("id");
+ALTER TABLE topology_source ADD CONSTRAINT "fk_topology_source_topology_version" FOREIGN KEY ("versionId") REFERENCES topology_version("id");
+ALTER TABLE topology_source_stream_mapping ADD CONSTRAINT "fk_topology_source_stream_mapping_topology_source" FOREIGN KEY ("sourceId", "versionId") REFERENCES topology_source("id", "versionId");
+ALTER TABLE topology_source_stream_mapping ADD CONSTRAINT "fk_topology_source_stream_mapping_topology_stream" FOREIGN KEY ("streamId", "versionId") REFERENCES topology_stream("id", "versionId");
+ALTER TABLE topology_sink ADD CONSTRAINT "fk_topology_sink_topology_version" FOREIGN KEY ("versionId") REFERENCES topology_version("id");
+ALTER TABLE topology_processor ADD CONSTRAINT "fk_topology_processor_topology_version" FOREIGN KEY ("versionId") REFERENCES topology_version("id");
+ALTER TABLE topology_processor_stream_mapping ADD CONSTRAINT "fk_topology_processor_stream_mapping_topology_processor" FOREIGN KEY ("processorId", "versionId") REFERENCES topology_processor("id", "versionId");
+ALTER TABLE topology_edge ADD CONSTRAINT "fk_topology_edge_topology_version" FOREIGN KEY ("versionId") REFERENCES topology_version("id");
```

Comment:
Seems like fk constraint  "FOREIGN KEY (streamId, versionId) REFERENCES topology_stream(id, versionId)" for topology_processor_stream_mapping is missing here.

Reply:
Nice finding. Will add.